pkgs/by-name/si/sirula/package.nix

```
@@ -1,36 +1,27 @@
-{
-  lib,
-  fetchFromGitHub,
-  rustPlatform,
-  pkg-config,
-  gtk3,
-  gtk-layer-shell,
-}:
+{ lib, fetchFromGitHub, rustPlatform, pkg-config, gtk3, gtk-layer-shell, }:
 
 rustPlatform.buildRustPackage rec {
   pname = "sirula";
-  version = "1.0.0-unstable-2023-09-02";
+  version = "1.1.0";
 
   src = fetchFromGitHub {
     owner = "DorianRudolph";
     repo = pname;
-    rev = "b15efe85ef1fe50849a33e5919d53d05f4f66090";
-    hash = "sha256-S0WbqY49nKaBUMWfgDKZxFLJuk7uFcnTfV8s86V0Zxs=";
+    rev = "f4da8f5fe50c367e7684334c31ec65fc0fe6997c";
+    hash = "sha256-rBaH2cIIaRoaw8Os60s4MknZywzDuGLagJiAvEYU4m8=";
   };
 
   cargoLock = {
     lockFile = ./Cargo.lock;
     outputHashes = {
-      "osstrtools-0.2.2" = "sha256-Co4pcikfN4vtIVK7ZsRGCWMAhMJWNNVZe/AdN1nMlmQ=";
+      "osstrtools-0.2.2" =
+        "sha256-Co4pcikfN4vtIVK7ZsRGCWMAhMJWNNVZe/AdN1nMlmQ=";
     };
   };
 
   nativeBuildInputs = [ pkg-config ];
 
-  buildInputs = [
-    gtk3
-    gtk-layer-shell
-  ];
+  buildInputs = [ gtk3 gtk-layer-shell ];
 
   meta = with lib; {
```

```suggestion 
meta = {
    description = "Simple app launcher for wayland written in rust";
    homepage = "https://github.com/DorianRudolph/sirula";
    license = [ lib.licenses.gpl3Plus ];
    maintainers = [ ];
    platforms = [ lib.platforms.linux ];
  };
}
```
Use of `with lib;` is discouraged. 
Also, do you mind adpoting this package (possibly later)?

---

Related resource: https://github.com/NixOS/nixpkgs/issues/208242